pkg/prometheus/promcfg.go

```diff
@@ -116,6 +116,34 @@ func addTLStoYaml(cfg yaml.MapSlice, namespace string, tls *v1.TLSConfig) yaml.M
 	return cfg
 }
 
+func addSafeAuthorizationToYaml(cfg yaml.MapSlice, version semver.Version, assetStoreKey string, store *assets.Store, auth *v1.SafeAuthorization) yaml.MapSlice {
+	if auth == nil || !version.GTE(semver.MustParse("2.26.0")) {
```

Comment:
(nit) easier to read?

```suggestion
	if auth == nil || version.LT(semver.MustParse("2.26.0")) {
```